src/components/implementation/fork_tests/micro_fork/micro_fork.c

```diff
@@ -7,25 +7,37 @@
 #define ITER (1024)
 u64_t meas[ITER];
 
+/*
+ * Warning - test assumes Composite can run at least ITER components.
+ * (And this may not take into account other components for this test
+ */
+
 void cos_init(void)
 {
 	u64_t start, end, avg, tot = 0, dev = 0;
 	int i, j;
 	spdid_t f;
 	int iter = ITER;
 
-	call();			/* get stack */
+	call();			/* get stack */ // how does this get the stack?
 	f = quarantine_fork(cos_spd_id(), cos_spd_id()-1); /* dirty way to find pong */
 	if (f == 0) printc("Error: initial fork failed\n");
```

Comment:
I haven't looked into the implementation for `quarantine_fork` to confirm, but normally errors are `< 0`.